log-viewer-cli/bin/etc/linux-desktop/log-viewer.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
We can add `create-desktop-icon.sh` script that creates `~/.local/share/applications/log-viewer.desktop` or `/usr/local/share/applications/log-viewer.desktop`. It could be more usable than reading the documentation and coping 'log-viewer.desktop' with replacing the path.